src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,139 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private Object[] objects;
+    private int size;
+
+    public ArrayList() {
+        size = 0;
+        objects = new Object[10];
+    }
```

Comment:
Magic numbers should be constants. The initial capacity '10' should be extracted into a constant with a clear name, such as 'DEFAULT_CAPACITY'.